docs/SRE_request_studio_content_api_addition

```diff
@@ -0,0 +1,152 @@
+# tl;dr
```

Comment:
Please affix the file name with `.md` so this can be previewed better

Reply:
OK